documentation/book/proc-configuring-kafka-mirror-maker-abort-on-send-failure.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-abort-on-send-failure.adoc
+
+[id='proc-configuring-kafka-mirror-maker-abort-on-send-failure-{context}']
+= Configuring the Mirror Maker to exit on a failed send
+
+Configure the mirror maker to exit on a failed send using the `KafkaMirrorMaker.spec.producer.abortOnSendFailure` property.
```

Comment:
you are saying "....to exit on a failed send...." but then than example set it to "false" so ... no exit :-)